lbz/consts.py

```diff
@@ -0,0 +1 @@
+DIRECT_LAMBDA_REQUEST = "direct_lambda_request"
```

Comment:
I would use the full name of this module: `constants.py` - there is no benefit to omitting these three letters 😉

Reply:
module has been removed